helix-term/src/commands.rs

```diff
@@ -2450,6 +2451,64 @@ fn jumplist_picker(cx: &mut Context) {
     cx.push_layer(Box::new(overlayed(picker)));
 }
 
+#[cfg(not(feature = "git"))]
+fn git_status_picker(cx: &mut Context) {
+    cx.editor
+        .set_error("Helix was built without the \"git\" feature");
+}
+
+#[cfg(feature = "git")]
+fn git_status_picker(cx: &mut Context) {
+    use helix_vcs::Git;
+
+    let proj_root = find_root(None, &[]);
```

Comment:
This is unnecessary. Gitoxide handles git repo detection automatically and more accurately. Just passing the cwd to gitoxide is sufficent

Reply:
Fixed in the latest push. It just uses cwd now.